zingolib/src/wallet/confirmation_status.rs

```diff
@@ -29,13 +36,13 @@ impl ConfirmationStatus {
     pub fn is_confirmed(&self) -> bool {
         matches!(self, Self::ConfirmedOnChain(_))
     }
-    pub fn is_confirmed_after_or_at(&self, height: &BlockHeight) -> bool {
+    pub fn is_confirmed_at_or_above(&self, height: &BlockHeight) -> bool {
```

Comment:
What's the rationale for having these predicates structured in this way?  I kind of like the idea that if one's true that the other's false?

I renamed these using the height metaphor since that's consistent with the "BlockHeight" nym.